angular-spa-ui/js/search-app/bootstrap.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = function(app){
```

Comment:
Setup codestyle in your IDE. I do not see the space character after "(app)" and before "{".